File: php/src/Advertisements/CivicCenter/Domain/CivicCenter.php
<?php
declare(strict_types=1);

namespace Demo\App\Advertisements\CivicCenter\Domain;

use Demo\App\Advertisements\CivicCenter\Domain\ValueObjects\CivicCenterName;
use Demo\App\Advertisements\Shared\ValueObjects\CivicCenterId;

final class CivicCenter
{
    public function __construct(private readonly CivicCenterId $id, private CivicCenterName $name)
    {
    }

    public function id(): CivicCenterId
    {
        return $this->id;
    }

    public function name(): CivicCenterName
    {
        return $this->name;
    }
}

File: kotlin/src/main/kotlin/advertisements/civiccenter/domain/CivicCenter.kt
package advertisements.civiccenter.domain

import advertisements.shared.value_object.CivicCenterId
import advertisements.civiccenter.domain.value_object.CivicCenterName

class CivicCenter(
    val id: CivicCenterId,
    var name: CivicCenterName
) {
    fun id(): CivicCenterId {
        return id
    }

    fun name(): CivicCenterName {
        return name
    }
}